src/test/ut/ut_cortxfs_xattr_ops.c

```diff
@@ -408,11 +408,16 @@ int listxattr_test_setup(void **state)
 	struct ut_xattr_env *ut_xattr_obj = XATTR_ENV_FROM_STATE(state);
 	struct ut_cfs_params *ut_cfs_obj = &ut_xattr_obj->ut_cfs_obj;
 
-	ut_xattr_obj->xattr[0] = file_name;
+
+	ut_xattr_obj->xattr = malloc(sizeof(char *)*xattr_set_cnt+1);
+	ut_xattr_obj->xattr[0] = malloc(sizeof(char *) * (strlen(file_name)+1));
+	memcpy(ut_xattr_obj->xattr[0], file_name, (strlen(file_name)+1));
 	for (i = 0; i<xattr_set_cnt; i++) {
-		ut_xattr_obj->xattr[i+1] = xattr_name[i];
+		ut_xattr_obj->xattr[i+1] = malloc(sizeof(char *) * (strlen(xattr_name[i])+1));
```

Comment:
Should be sizeof (char) instead of sizeof (char*).

Reply:
fixed